File: Semana 1/9. Compara dos números e imprime el resultado.py
num1 = int(input("Ingrese el primer numero: "))
num2 = int(input("Ingrese el segundo numero: "))

if num1 > num2:
    print(f"{num1} es mayor a {num2}")
elif num2 > num1:
    print (f"{num2} es mayor a {num1}")
else:
    print("Ambos numeros son iguales")

File: Semana 1/9. Compara dos números e imprime el resultado.cpp
#include <iostream>
using namespace std;

int main() {
    int num1;
    int num2;

    cout << "Ingrese el primer numero: ";
    cin >> num1;

    cout << "Ingrese el segundo numero: ";
    cin >> num2;

    if (num1 > num2) {
        cout << "El numero " << num1 << " es mayor que " << num2;
    } else if (num1 < num2) {
        cout << "El numero " << num1 << " es menor que " << num2;
    } else {
        cout << "Ambos numeros son iguales";
    }

    return 0;
}
